easybuild/easyblocks/p/psi.py

```diff
@@ -98,7 +98,8 @@ def configure_step(self):
             raise EasyBuildError("Boost module not loaded.")
 
         # pre 4.0b5, they were using autotools, on newer it's CMake
-        if LooseVersion(self.version) <= LooseVersion("4.0b5"):
+        if not os.path.exists(os.path.join(self.builddir, self.psi_srcdir, "CMakeLists.txt")):
+            self.log.info("Using configure based build")
```

Comment:
This looks OK, but it causes problems with existing PSI easyconfigs, in case they already include `CMakeLists.txt`, but were still being installed with `./configure`, and thus don't include `CMake` as a (build) dependency.

So, this introduces a regression of sorts...

I think you need to keep the version check in place, but make an exception for installations where `PSI4` is the software name?